Sources/SectionScrubber.swift

```diff
@@ -215,10 +206,9 @@ public class SectionScrubber: UIView {
             if newYPosForSectionScrubber >= self.containingViewFrame.size.height + self.containingViewFrame.minY - bottomBorderOffset {
                 newYPosForSectionScrubber = self.containingViewFrame.size.height + self.containingViewFrame.minY - bottomBorderOffset
 
-                let centerPoint = CGPoint(x: self.center.x + self.collectionView.contentOffset.x, y: self.collectionView.contentSize.height - (self.viewHeight / 2) - bottomBorderOffset);
-                print(centerPoint)
+                let extraMargin = UIScreen.mainScreen().bounds.height - self.containingViewFrame.height
```

Comment:
Ah cool! So this compensates for the scrubber not being able to go all the way down?